# This is a message to communicate the agent's prediction
Header header
float64[] SSx
float64[] SSy
